source/MaterialXCore/Interface.cpp

```diff
@@ -301,6 +306,26 @@ NodePtr Input::getConnectedNode() const
     return PortElement::getConnectedNode();
 }
 
+void Input::setInterfaceInput(const string& interfaceName)
+{
+    if (interfaceName != EMPTY_STRING)
```

Comment:
Minor: I'd recommend expressing this as `!interfaceName.empty()` for consistency and efficiency.